reference/3.0/Microsoft.PowerShell.Core/About/about_Registry_Provider.md

```diff
@@ -504,18 +531,18 @@ This parameter works as designed on the [Set-ItemProperty](../../Microsoft.Power
 ## Using the pipeline
 
 Provider cmdlets accept pipeline input. You can use the pipeline to simplify
-task by sending provider data from one cmdlet to another provider cmdlet.
-To read more about how to use the pipeline with provider cmdlets, see the
-cmdlet references provided throughout this article.
+task by sending provider data from one cmdlet to another provider cmdlet. To
+read more about how to use the pipeline with provider cmdlets, see the cmdlet
+references provided throughout this article.
 
 ## Getting help
 
 Beginning in Windows PowerShell 3.0, you can get customized help topics for
 provider cmdlets that explain how those cmdlets behave in a file system drive.
 
 To get the help topics that are customized for the file system drive, run a
-[Get-Help](../Get-Help.md) command in a file system drive or use the `-Path`
-parameter of [Get-Help](../Get-Help.md) to specify a file system drive.
+`Get-Help` command in a file system drive or use the `-Path` parameter to
```

Comment:
I'd change `-Path` to **Path** to be more consistent with parameter references.

Reply:
done